ansible/configs/sap-integration/files/pvc-registry.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
what are you using this for?

With the version of OpenShift you are using, you don't need this when using IPI.

Reply:
done